docs/book/src/topics/multitenancy.md

```diff
@@ -213,6 +213,9 @@ There are multiple AWS assume role permissions that need to be configured in ord
   }
   ```
 
+Both of these permissions can be enabled via clusterawsadm as documented [here](usingcluster-awsadm-to-fulfill-prerequisites.md#cross-account-role-assumption).
```

Comment:
This link will not work. You have to link to the actual doc.

Reply:
@Skarlso I dont understand why this wouldnt work.  https://github.com/kubernetes-sigs/cluster-api-provider-aws/blame/main/docs/book/src/topics/using-clusterawsadm-to-fulfill-prerequisites.md#L74 works and its a link relative to the topics directory.  Or are you saying that #label cant work?